src/components/Connect/Metamask/MetamaskConnect.tsx

```diff
@@ -67,6 +67,25 @@ const MetamaskConnect = ({ onConnect }: MetamaskConnectProps) => {
     }
   };
 
+  /**
+   * Clicking my connect to metamask button
+   */
+  const connect = async (): Promise<void> => {
+    try {
+      setLoading(true);
```

Comment:
Loading is not set back to false anywhere.

Reply:
The login process is divided into 2 parts due to the modal window. The state changes to false when the modal window is closed or after entering the password. Lines 66, 113